Source/CarthageKit/Errors.swift

```diff
@@ -188,6 +191,9 @@ extension CarthageError: Printable {
 		case let .InvalidArchitectures(description):
 			return "Invalid architecture: \(description)"
 
+		case let .InvalidUUIDs(description):
+			return "Invalid UUIDs: \(description)"
```

Comment:
The description printed is specified at https://github.com/Carthage/Carthage/pull/684/files#diff-f66644e8cfbcad3c3826395a690ddfbcR1307 — I'll beef it up a little.